packages/patternfly-4/package.json

```diff
@@ -5,12 +5,13 @@
   "author": "Red Hat",
   "private": true,
   "scripts": {
-    "build": "yarn compile-examples-css && node --max_old_space_size=4096 ./node_modules/.bin/gatsby build --prefix-paths",
```

Comment:
Why shouldn't this be a part of the build step?

Reply:
it will update an existing file that is already checked in. How does the build handle new/updated uncommited changes?